README.md

```diff
@@ -1,5 +1,10 @@
 ﻿# Parts Unlimited
 
+[![Build Status](https://dev.azure.com/rudiansen-lab/Parts%20Unlimited%20Demo/_apis/build/status/PartsUnlimited-CI?branchName=master)](https://dev.azure.com/rudiansen-lab/Parts%20Unlimited%20Demo/_build/latest?definitionId=9&branchName=master)
+[![Board Status](https://dev.azure.com/rudiansen-lab/3710fb5f-9603-4b82-aeb1-38fc03e3101c/e4881f68-9c92-4db5-8395-c9e986c9bb03/_apis/work/boardbadge/ac49a16e-e1e6-409f-8a99-0fb1a85e943c?columnOptions=1)](https://dev.azure.com/rudiansen-lab/3710fb5f-9603-4b82-aeb1-38fc03e3101c/_boards/board/t/e4881f68-9c92-4db5-8395-c9e986c9bb03/Microsoft.RequirementCategory/)
+
```

Comment:
@rudiansen Should we add a new line break here?

Reply:
Yeah, just for eye looking.